intervaltree/interval.py

```diff
@@ -53,6 +53,31 @@ def overlaps(self, begin, end=None):
         except:
             return self.contains_point(begin)
 
+    def overlap_size(self, begin, end=None):
+        """
+        Return the overlap size between two intervals or a point
+        :param begin: beginning point of the range, or the point, or an Interval
+        :param end: end point of the range. Optional if not testing ranges.
+        :return: Return the overlap size, None if not overlap is found
+        :rtype: int
```

Comment:
We might not always return an int. We could also be returning a real, a time duration, or some other distance type.

Reply:
changed the doc